CodeGeneration/Sources/SyntaxSupport/Node.swift

```diff
@@ -118,14 +118,14 @@ public class Node {
           let unexpectedDeprecatedName: String?
 
           if i == 0 {
-            unexpectedName = "UnexpectedBefore\(child.name)"
-            unexpectedDeprecatedName = child.deprecatedName.map { "UnexpectedBefore\($0)" }
+            unexpectedName = "UnexpectedBefore\(child.type)"
```

Comment:
I would prefer to do `child.name.withFirstLetterUppercased` when building the unexpected name. I know that the result is the same as if we used the type name, but that’s more of a coincidence right now. What we’re intending to do here is to stitch the names together and the code should reflect that in my opinion. Otherwise every time I read this, I would wonder: What has the type name to do with the property name here.

Reply:
I fully agree with you. I guess I was trying too hard to change the code base in order to make `child.name` private.

Reply:
Done